### ISSUE TITLE
Sensors calibration

### ISSUE THREAD

**Issue by AMIRAMIM** [KevinLADLee/carla_dataset_tools#10]
Hi, 

I am interested in knowing whether the sensors used for the recording process are calibrated ? Thank you

**Comment by KevinLADLee** [comment]
Yep, the extrinsic parameters correspond to the sensor_config_json. Your own camera parameters could calculate the intrinsic parameter, just like https://github.com/KevinLADLee/carla_dataset_tools/blob/master/recorder/camera.py#L60.
